### ISSUE TITLE
why not update alpha?

### ISSUE THREAD

**Issue by scirocc** [CoderAT13/BipedalWalkerHardcore-SAC#1]
#alpha_loss = -(self.log_alpha * (log_pi + self.target_entropy).detach()).mean()
#self.alpha_optim.zero_grad()
#alpha_loss.backward()
#self.alpha_optim.step()
 why not update?

**Comment by CoderAT13** [comment]
it's a experiment to test whether it is better to use fixed alpha, because I found a result in the SAC paper showing that it might be better to use fixed alpha if alpha is good.
